browse.php

```diff
@@ -248,12 +248,12 @@ class="form-submit">
                                 <h4 id="clock<?php echo $row['StockItemID'] ?>" style="font-weight: bold;"></h4>
                                 <h2 class="StockItemPriceText">
                                     <s class="strikedtext">
-                                        <?php echo calculatePriceBTW($row['SellPrice'], $row['TaxRate']); ?>
+                                        <?php echo calculatePriceBTW($row['RecommendedRetailPrice'], $row['TaxRate']); ?>
```

Comment:
Waarom pak je hier de calculatePriceBTW, was het niet zo dat de SellPrice al het kloppende bedrag is?

Reply:
SellPrice in browse.php heeft de btw er inderdaad bij gerekend, maar dat wordt verder nergens aangetoond. Omdat op alle andere plekken er calculatePriceBTW() wordt gebruikt wilde ik gaan voor consistentie.